internal/pkg/storage/db/storage.go

```diff
@@ -233,6 +231,33 @@ func (s *Storage) Count() int {
 	return count
 }
 
+// GetUniqueUsers return slice of unique user ID
+func (s *Storage) GetUniqueUsers(ctx context.Context) []int {
+	var users []int
+	rows, err := s.connection.DB.QueryContext(ctx, `
+        SELECT user_id
```

Comment:
```suggestion
        SELECT count(distinct user_id)
```
Можно вместо DISTINCT, как и в текущей реализации, использовать GROUP BY. но в любом случае сумму значительно быстрее и дешевле получить на уровне запроса к БД.